code/game/objects/items/devices/modkit.dm

```diff
@@ -111,6 +111,20 @@
 	original[1] = /obj/item/clothing/suit/space/rig/security
 	finished[1] = /obj/item/clothing/suit/space/rig/security/fat
 
+/obj/item/device/modkit/syndi_commander
+	name = "syndicate commander hardsuit modification kit"
+	desc = "For showing who's the boss. Apply to hardsuit."
+
+/obj/item/device/modkit/syndi_commander/New()
+	..()
+	parts = new/list(1)
+	original = new/list(1)
+	finished = new/list(1)
```

Comment:
you can do all these as new/list(1 = "/obj/item/clothing/suit/space/rig/syndi/commander")

Reply:
`new` isn't needed in this context, also I'm not sure if the `1 = typepath` is really necessary

Reply:
i hate oldcoders so much

Reply:
fix shitcode

Reply:
why did you guys bitch at me and not at kurf when he did the exact same thing in #24969 😔

Reply:
This whole file needs a nuclear genocide.